### ISSUE TITLE
508-defect-3 [SCREENREADER]: STEM Standalone - Links that open in new tab do not alert user that a new tab will be opened

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#18247]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

Noah Gelman

**VFS Point of Contact:** _First name only. Delete if not applicable._

## User Story or Problem Statement

As a screen reader who cannot see and uses the keyboard to navigate, when tabbing to various links in the form, they do not announce to the user that they open in a new tab. User might think the links will exit the user from the form.

## Details

Screen reader announcement for links that open to new tabs should announce that they open in a new tab

## Acceptance Criteria

- [ ] Add hidden screen reader only text to link text that announces it opening in a new window or give link aria-label that includes text that announces opening in a new window

## Steps to Recreate

https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/applicant/information

Navigate the form as normal. Various sections have links that open in new window

## WCAG or Vendor Guidance (optional)

https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html

**Comment by ksantiagomorgan6** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/15604

**Comment by delli-gatti-michael** [comment]
Local testing complete

**Comment by delli-gatti-michael** [comment]
@noahgelman This is on Staging and ready for your review

**Comment by noahgelman** [comment]
@delli-gatti-michael Approved
